### PR TITLE
[TRIVIAL] Set unfunded offers fix switch date

### PR DESCRIPTION
**Pull request by seelabs**
@HowardHinnant Please dummy check the date

### CODE REVIEW

File: src/ripple/ledger/impl/View.cpp [hunk 1]
```diff
@@ -61,9 +61,8 @@ bool amendmentRIPD1274 (NetClock::time_point const closeTime)
 NetClock::time_point const& amendmentRIPD1298SoTime ()
 {
     using namespace std::chrono_literals;
-    // TBD Move this date up when we know the release date
-    // Thu Jan 5, 2017 10:00:00am PDT
-    static NetClock::time_point const soTime{536954400s};
+    // Fri Nov 18, 2016 10:00:00am PDT
+    static NetClock::time_point const soTime{532807200s};
```
Comment (HowardHinnant) [inline src/ripple/ledger/impl/View.cpp]: Nitpick:  The correct abbreviation is PST.  Otherwise checks out.

**Comment by HowardHinnant** [comment]
👍  One nitpicky comment about a comment. :-)

File: src/ripple/ledger/impl/View.cpp (hunk 1 above)
Comment (seelabs) [inline src/ripple/ledger/impl/View.cpp]: Fixed. Thanks.

**Comment by nbougalis** [comment]
Merged as 665ad180cbe3ab9b1913a16a9d0baa378a6ad377

**Pull request closed without merging**